package/MDAnalysis/core/Selection.py

```diff
@@ -769,38 +741,6 @@ def __repr__(self):
         return "<'NucleicSugarSelection' >"
 
 
-class CASelection(BackboneSelection):
-    """Select atoms named CA in protein residues (supposed to be the C-alphas)
-    """
-
-    def _apply(self, group):
-        return set([a for a in group.atoms if (a.name == "CA" and a.resname in self.prot_res)])
-
-    def __repr__(self):
-        return "<'CASelection' >"
-
-
-class BondedSelection(Selection):
```

Comment:
Bonded does not even work with the new topology system? I think we would *like* to have this working. Can you please raise a new issue to reimplement + test?